server/sonar-db-dao/src/main/java/org/sonar/db/rule/RuleDao.java

```diff
@@ -47,11 +47,11 @@
   public Optional<RuleDto> selectByKey(DbSession session, String organizationUuid, RuleKey key) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)